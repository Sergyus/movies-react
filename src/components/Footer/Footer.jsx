import React from 'react';
import styles from './Footer.scss';

const Footer = () => (
  <footer className={styles.footer}>
    <h2 className={styles.logo}>Films</h2>
    <p className={styles.copyright}>Copyright Â© 2019 FILMS. DZMITRY</p>
  </footer>
);

export default Footer;
