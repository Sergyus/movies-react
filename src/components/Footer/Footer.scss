@import "../../styles/variables.scss";

.footer {
  text-align: center;
  padding: 40px 0;
}

.logo {
  color: $blue-color;
  text-transform: uppercase;
  font-size: 1.7rem;
  letter-spacing: 3px;
  margin-bottom: 30px;
  font-weight: normal;
}

.copyright {
  color: $gray-color;
}
