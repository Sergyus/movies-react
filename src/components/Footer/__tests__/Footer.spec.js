import React from 'react';
import { create } from 'react-test-renderer';
import Footer from '..';

describe('Footer component renders correctly', () => {
  it('Footer renders correctly', () => {
    const tree = create(<Footer />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
