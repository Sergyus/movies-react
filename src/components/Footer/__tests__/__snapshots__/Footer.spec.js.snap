// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component renders correctly Footer renders correctly 1`] = `
<footer
  className="footer"
>
  <h2
    className="logo"
  >
    Films
  </h2>
  <p
    className="copyright"
  >
    Copyright Â© 2019 FILMS. DZMITRY
  </p>
</footer>
`;
